app-eselect/eselect-luajit/metadata.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>ilmostro7@gmail.com</email>
+		<name>ILMostro</name>
+	</maintainer>
```

Comment:
You probably need to add proxy-maint.

Reply:
ahh, yeah; forgot about that.  Thank you